cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -197,24 +196,3 @@ def test_publishing_a_page(self):
         latest_content = json.loads(latest.content_json)
         assert latest_content['shared']
         assert latest_content['live']
-
-
```

Comment:
Removing these tests since all they really do is test the logic of `if is_live and settings.ENABLE_AKAMAI_CACHE_PURGE` and not the akamai purge itself. I will try to come up with another way of testing this, particularly when I add the check for if it's a new page.